### PR TITLE
feat: added About page

### PR DESCRIPTION
**Pull request by elena-ambitska**
Added 'About' page

<details> 

- desktop view  

image

- mobile view  

image

</details>

### CODE REVIEW

File: homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-<body>
-  <main class="main">
-    <h1 class="main__heading--h1">About Me</h1>
-    <p class="main__paragraf">Hi! My name is Andrii Morozov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.</p>
-    <p class="main__paragraf">This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it" class="link--red" target="_blank">'Frontend for beginners' from Masters Academy in 2023.</a></p>
-    <p class="main__paragraf">This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio. </p>
-    <p class="main__paragraf">You can contact me via <a href="https://www.facebook.com/profile.php?id=100024573128932" class="link--red" target="_blank">facebook</a> and/or check out my GitHub.</p>
-  </main>
-</body>
+    <head>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html:3]: These are the changes of your colleague, not your. Please remove it from your pull request.

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/style.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  max-height: 356px;
+  width: 100%;
+  max-width: 536px;
+  padding-bottom: 24px;
+}
+
+.section__title {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 1.2;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/GamesProject/style.css:29]: The result `line-height` is different than on the design, but, proly, your way of setting it is more correct.

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/style.css (hunk 2 above)
Comment (vladyslav-yermolin-moc) [reply homeworks/olena.ambitska_elena-ambitska/GamesProject/style.css:29]: So, there is no need to change your `1.2` value

File: homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html (hunk 1 above)
Comment (elena-ambitska) [reply homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html:3]: @vladyslav-yermolin-moc I made a new branch with chery-picked commits.
Check pls [pull request](https://github.com/MastersAcademy/fe-course-basic-2023/pull/92)

Comment (vladyslav-yermolin-moc) [reply homeworks/andrii.morozov_AndriiMorozov88/GamesProject/index.html:3]: Ok, so I'm closing this pr

**Pull request closed without merging**